docs/architecture/devops-for-aspnet-developers/actions-codeql.md

```diff
@@ -0,0 +1,183 @@
+---
+title: DevOps with .NET and GitHub Actions - Secure code with CodeQL
+description: Add security scanning to your .NET code with GitHub Actions and CodeQL
+author: colindembovsky
+ms.date: 03/04/2021
+---
+
+# Secure .NET Code with CodeQL and GitHub Actions
+
+[CodeQL](https://codeql.github.com/docs/codeql-overview/about-codeql/) is a static code analysis engine that can automate security and quality checks. With CodeQL, you can perform *variant analysis*, which uses known vulnerabilities as seeds to find similar issues. CodeQL is part of [GitHub Advanced Security](https://docs.github.com/github/getting-started-with-github/about-github-advanced-security) that includes:
+
+> [!div class="checklist"]
+
+> * Code scanning&mdash;find potential security vulnerabilities in your code.
+> * Secret scanning&mdash;detect secrets and tokens that are committed.
+> * Dependency scanning&mdash;detect vulnerabilities in packages that you consume.
```

Comment:
Same as above. Want to confirm if this renders correctly on staging, or otherwise add `>` in line 13

Reply:
I think it looks good.

![image](https://user-images.githubusercontent.com/13281246/114818374-52599a00-9dd9-11eb-8e69-be70888b9b97.png)

Reply:
![image](https://user-images.githubusercontent.com/13281246/114836053-de76bc00-9def-11eb-9507-965101dd0f02.png)